cmd/tink-worker/worker/worker.go

```diff
@@ -376,14 +425,15 @@ func isLastAction(wfContext *pb.WorkflowContext, actions *pb.WorkflowActionList)
 }
 
 func (w *Worker) reportActionStatus(ctx context.Context, actionStatus *pb.WorkflowActionStatus) error {
-	l := w.logger.With("workflowID", actionStatus.GetWorkflowId,
+	l := w.getLogger(ctx).With("workflowID", actionStatus.GetWorkflowId(),
 		"workerID", actionStatus.GetWorkerId(),
 		"actionName", actionStatus.GetActionName(),
 		"taskName", actionStatus.GetTaskName(),
 	)
 	var err error
 	for r := 1; r <= w.retries; r++ {
-		_, err = w.client.ReportActionStatus(ctx, actionStatus)
+		l.Info("Reporting Action Status")
```

Comment:
log messages should not be `Sentence case`. I thought we had a linter to catch this :thinking: (maybe needs a lint-install update). Would be nice not to introduce cases where we'd need to fix later on.

Reply:
fixed